plugins/obs-ffmpeg/texture-amf.cpp

```diff
@@ -1008,6 +1009,9 @@ static obs_properties_t *amf_properties_internal(bool hevc)
 	obs_property_set_long_description(p,
 					  obs_module_text("AMFOpts.ToolTip"));
 
+	p = obs_properties_add_bool(props, "repeat_headers", "repeat_headers");
+	obs_property_set_visible(p, false);
+
```

Comment:
The property is not required, `obs_data` is enough. The PR should work without it.

Reply:
Resolved per off-thread request.